server/socket.js

```diff
@@ -50,6 +51,7 @@ export default function startSocketServer(http) {
 
 
     socket.on(CS.JOIN_TO_CHANNEL, channelId => {
+      if (!validate(channelId).isString().inRange(20, 30).end()) return;
```

Comment:
This should be a 24 degit hex id: `^[0-9a-fA-F]{24}$`, like this. Same for validating all other ids.